src/accelerate/big_modeling.py

```diff
@@ -100,6 +101,7 @@ def init_on_device(device: torch.device, include_buffers: bool = False):
     ```
     """
     # TODO(shingjan): remove the torch version check once older versions are deprecated
+    include_buffers = parse_flag_from_env('INIT_INCLUDE_BUFFERS', False)
```

Comment:
Let's just set the default (at line 83) to `None` and then do this only if `include_buffers` is `None` (so the user can override the default from the env variable).

Also please leave the comment after this addition (before the torch version test).

Reply:
Let's also add `ACCELERATE_` as a prefix please so we can keep consistency with the rest of the library :)

Reply:
Thanks @sgugger and @muellerzr for the review! I also make the same change to default value of `include_buffers` for `init_empty_weights` to make the env variable useful and keep BC. Thanks!